corehq/apps/sms/tasks.py

```diff
@@ -604,3 +610,25 @@ def queued_sms():
 
 
 datadog_gauge_task('commcare.sms.queued', queued_sms, run_every=crontab())
+
+if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
+    @periodic_task(run_every=crontab(day_of_month='2', minute=0, hour=0))
+    def send_monthly_sms_report():
+        start_date = date.today().replace(day=1) - relativedelta(months=1)
+        _, last_day = calendar.monthrange(start_date.year, start_date.month)
+        end_date = start_date.replace(day=last_day)
+        filename = call_command('get_icds_sms_usage', 'icds-cas', str(start_date), str(end_date))
```

Comment:
What happens if this command fails? Is it possible to notify us?

Reply:
It would make it to sentry and people who should get the email, won't.
But good point to still notify, going to notify the same recipients. 👍

Reply:
https://github.com/dimagi/commcare-hq/pull/25882/commits/4eeee89f818ae90291f65b65785640fb724745a5